LibGAP.jl/src/gap.jl

```diff
@@ -37,16 +38,37 @@ Base.setproperty!(x::MPtr, f::Symbol, v) = GAP.GAPFuncs.ASS_REC(x, RNamObj(f), t
 
 import Base: *, +, -, /, ^, mod, <, ==
 
-#
-+(x::GAPInputType, y::GAPInputType) = GAP.GAPFuncs.SUM(x, y)
--(x::GAPInputType, y::GAPInputType) = GAP.GAPFuncs.DIFF(x, y)
-*(x::GAPInputType, y::GAPInputType) = GAP.GAPFuncs.PROD(x, y)
-/(x::GAPInputType, y::GAPInputType) = GAP.GAPFuncs.QUO(x, y)
-^(x::GAPInputType, y::GAPInputType) = GAP.GAPFuncs.POW(x, y)
-mod(x::GAPInputType, y::GAPInputType) = GAP.GAPFuncs.MOD(x, y)
+typecombinations = [[:GAPInputType_noint,:GAPInputType_noint],
+                    [:GAPInputType_noint,:Int64],
+                    [:Int64,:GAPInputType_noint]]
```

Comment:
Warum reichen nicht zwei Combos: `:GAPInputType_noint, :GAPInputType]` und `[:GAPInputType, :GAPInputType_noint]`?

Reply:
Because this would give several (equal) options for the two types in `:GAPInputType_noint`. This way, we only have one possibility for each type combination